code/modules/awaymissions/mission_code/moonoutpost19.dm

```diff
@@ -92,7 +92,7 @@
 
 /obj/item/paper/fluff/awaymissions/moonoutpost19/engineering
 	name = "Engineering Instructions"
-	info = "Alright, listen up. If you're reading this, I'm either taking a shit or I've been recalled back to Command. Either way, you'll need to know how to restore power. We've stolen this stuff from Nanotrasen, so all the equipment is jury-rigged. We have generators that work on both plasma and uranium, about 50 sheets should power the outpost for quite a while. If the generators aren't working, which is very likely, take the power cell on the desk and put it into the APC in the hallway. That should get the place running, at least for a little while."
+	info = "Alright, listen up. If you're reading this, I'm either taking a shit or I've been recalled back to Command. Either way, you'll need to know how to restore power. We've stolen this stuff from Nanotrasen, so all the equipment is barely operational. We have generators that work on both plasma and uranium, about 50 sheets should power the outpost for quite a while. If the generators aren't working, which is very likely, take the power cell on the desk and put it into the APC in the hallway. That should get the place running, at least for a little while."
```

Comment:
`We've stolen this stuff from Nanotrasen`

Reply:
It's a Gateway outpost thing. Yeah, I'm still thinking on keeping some Nanotrasen references since they'll be an existing (although hostile) company.